### DISCUSSION TITLE
Youtube not automatically clearing notifications

### DISCUSSION THREAD

**Discussion by TheFantasticWarrior** [ReVanced/revanced-discussions#932] [Question] [answered]
Not sure if it's a revanced issue or youtube issue, but I expect the 
notifications to clear after going to the notifications tab in app

probably related patches:
client-spoof
custom-branding
debugging
microg-support
predictive-back-gesture

**Comment by oSumAtrIX** [comment, marked as answer]
That's default behavior on YouTube or MicroG.

**Reply by TheFantasticWarrior** [reply]
Then can a patch be added for that?

**Reply by oSumAtrIX** [reply]
If it's the latter, no
